emission/net/ext_service/otp/otp.py

```diff
@@ -112,116 +107,19 @@ def turn_into_trip(self, _id, user_id, trip_id, is_fake=None):
         final_start_time = otp_time_to_ours(our_json['plan']['itineraries'][0]["startTime"])
         final_end_time = otp_time_to_ours(our_json['plan']['itineraries'][0]["endTime"])
         cost = 0
-
-        #print start_city, end_city
-        
-        if "RAIL" in mode_list or "SUBWAY" in mode_list:#
-            #print our_json
+        if "RAIL" in mode_list or "SUBWAY" in mode_list:
             try:
-                print our_json['plan']['itineraries'][0]['fare'].keys()
                 cost = float(our_json['plan']['itineraries'][0]['fare']['fare']['regular']['cents']) / 100.0   #gives fare in cents 
             except:
                 cost = 0
         elif "CAR" in mode_list:
-            #start_city_car = str(Geocoder().reverse_geocode(car_start_coordinates.get_lat(),car_start_coordinates.get_lon())).split(',')[1]
-            #end_city_car = str(Geocoder().reverse_geocode(car_end_coordinates.get_lat(), car_end_coordinates.get_lon())).split(',')[1]
-            #traffic_time = get_travel_time(start_city_car, end_city_car)
-            cost = calc_car_cost(car_dist)
-            #if traffic_time > final_end_time - final_start_time:
-            #    print "Driving is bad: " + traffic_time
-            #    final_end_time = final_start_time + traffic_time
-
+            # TODO calculate car cost
```

Comment:
Can you file an issue for this? I think that it might be more serious than just the TODO...